H/HiGHS/bundled/patches/v1.3.0.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index e5d3a14b8..8bd2dac90 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -125,8 +125,8 @@ enable_cxx_compiler_flag_if_supported("-Wno-format-truncation")
+ enable_cxx_compiler_flag_if_supported("-pedantic")
+ endif()
+ 
+-check_cxx_compiler_flag("-mpopcnt" FLAG_MPOPCNT_SUPPORTED)
+-set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -mpopcnt")
++# check_cxx_compiler_flag("-mpopcnt" FLAG_MPOPCNT_SUPPORTED)
++# set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -mpopcnt")
```

Comment:
@lgottwald why is `mpopcnt` needed here? It isn't portable across platforms.

Reply:
If anything, you have to do something like https://github.com/rdkit/rdkit/pull/5182/files (check [`CMAKE_SYSTEM_PROCESSOR`](https://cmake.org/cmake/help/latest/variable/CMAKE_SYSTEM_PROCESSOR.html))